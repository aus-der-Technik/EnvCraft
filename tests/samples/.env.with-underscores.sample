# Test variable name with underscores
USER_CREDENTIALS=
