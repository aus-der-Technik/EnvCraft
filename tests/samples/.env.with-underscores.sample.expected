USER_CREDENTIALS=myname:mypass
